src/main/io/vtx_control.h

```diff
@@ -28,7 +28,11 @@
 #include "pg/pg.h"
 #include "fc/rc_modes.h"
 
-#define MAX_CHANNEL_ACTIVATION_CONDITION_COUNT  10
+#ifndef MAX_VTX_CONDITIONS
+#define MAX_VTX_CONDITIONS  10
+#endif
+
+STATIC_ASSERT(MAX_VTX_CONDITIONS <= 99, "MAX_VTX_CONDITIONS must be <= 99");
```

Comment:
@nerdCopter probably you did not refresh the page, as this was final request :)
```suggestion
STATIC_ASSERT(MAX_VTX_CONDITIONS <= 99, "MAX_VTX_CONDITIONS must be <= 99");
```